### DISCUSSION TITLE
Play while sitting?

### DISCUSSION THREAD

**Discussion by Elleby77** [curif/AgeOfJoy-2022.1#84] [Ideas]
I just discovered this app and it is amazing. The only issue is that I am disabled, which means I have to play while sitting, which makes the monitors a bit too high for me. I tried the Quest's Accessibility option for height and that made the monitor the right height, however I couldn't bend down far enough to reach the coin slots.

Possible solutions I came up with:
- Make the height adjustable.   What might be ideal would be like sitting on a stool in front of the cabinet.
- Make the entire lower half of the cabinet front (below control panel) accept coins.
- Add a crouch button. A user could then crouch to insert a coin.

**Comment by curif** [comment]
Thank you for the suggestion.
Insert a coin by pressing the A button is ok for you? If not, the second one is quick to develop as well.

**Reply by Elleby77** [reply]
Yes, inserting a coin with A would be a solution as well.   However, some games seem to want you to insert a coin after the emulation has started,  so using A in that scenario would not work,   unless you make it so that it works when holding the coin.

**Reply by curif** [reply]
Yep. You are right.

May be a bigger area to insert the coin is better.

Thanks again

**Comment by WilsonBradley** [comment]
@Elleby77 Hi and welcome!!!
Yes, the limit position of the user to the cabinet is still being tweaked. Even from normal gameplay (this version) the user is not close enough. I believe the developer guru's are looking into this for next release.
Thanks for your input.

**Comment by curif** [comment]
Thanks.

**Comment by curif** [comment]
Issue #85
